services/gcp/cloudkms/ekmConnections.yml

```diff
@@ -0,0 +1,40 @@
+name: Cloud KMS EKM Connections
+description: >-
+  An EKM connection organizes VPC connections to your on-premises external key managers 
+  in a specific Google Cloud location. An EKM connection allows you to connect to and use keys 
+  from an external key manager over a VPC network
+scope: CRITICAL
+notes: >-
+  Cloud KMS is an extremely sensitive service. Keys can be used for encryption/decryption of 
+  sensitive data or for the creation or verification of digital signatures.
```

Comment:
Not sure repeating these notes is relevant to ekmConnections, consider removing or adding different notes.

Reply:
Yeah, copy-paste artifact, I removed. I also downgraded the scope to MEDIUM-- since the connection resource does not actually store any secret content I think it shouldn't be CRITICAL, that was also left over from copy-paste.